pkg/console/console.go

```diff
@@ -92,6 +93,23 @@ func (c *Console) setContentByName(name string, content string) error {
 	return err
 }
 
+func (c *Console) CloseElement(name string) error {
```

Comment:
Seems like we seldom handle the error in the caller. Can we log the error and do no error return for the function?

Reply:
yes